src/sage/schemes/hyperelliptic_curves/hyperelliptic_generic.py

```diff
@@ -38,7 +38,8 @@
 from sage.rings.power_series_ring import PowerSeriesRing
 from sage.rings.laurent_series_ring import LaurentSeriesRing
 from sage.rings.real_mpfr import RR
-from sage.functions.all import log
+from sage.misc.lazy_import import lazy_import
+lazy_import("sage.functions.all", "log")
 from sage.structure.category_object import normalize_names
```

Comment:
How about grouping `lazy_import` statement(s) separated from normal imports, as done in other files?

Reply:
Thanks, done in f52c4fe8c5e